cloudpickle/cloudpickle.py

```diff
@@ -278,6 +278,24 @@ def __init__(self, file, protocol=None):
         # map ids to dictionary. used to ensure that functions can share global env
         self.globals_ref = {}
 
+        # Functions belonging to the same module (the __main__ or a dynamic
+        # module) share the same global variables. Using cloudpickle, they will
+        # share those global variables in the process they are unpickled in as
+        # well. However, the state of those global variables may differ in the
+        # pickled functions, for example if one global variable was mutated
+        # between two calls to pickle.dump. override_existing_globals is a
+        # switch that allows two different behaviors:
+
+        # - If override_existing_globals=True, the global variables of the
+        # module containing this function will be overridden in the process
```

Comment:
"module containing the definition of the nested or dynamically defined function"